.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "tests/engine_tests/engine-test-data"]
 	path = tests/engine_tests/engine-test-data
-	url = git@github.com:Flagsmith/engine-test-data.git
+	url = https://github.com/Flagsmith/engine-test-data.git
```

Comment:
ssh-based submodule didn't play well with `pip` in GH pipelines in dowstream PRs (https://github.com/Flagsmith/flagsmith/pull/2290, https://github.com/Flagsmith/edge-api/pull/153 and https://github.com/Flagsmith/flagsmith-engine/pull/167).
This will be removed before merge.

Reply:
@matthewelwell encountered the same problem and we decided to leave the HTTPS submodule.